.vscode/settings.json

```diff
@@ -1,26 +0,0 @@
-{
```

Comment:
If this file doesn't prevent the changes from working (I believe it doesn't), its removal shouldn't be in this PR.